src/sagemaker/workflow/pipeline_context.py

```diff
@@ -93,13 +132,33 @@ def runnable_by_pipeline(run_func):
     The job will be started during pipeline execution.
     """
 
+    def _restructure_model_step_args(context: _ModelStepArguments):
+        """Restructure the pipeline session context arguments for a model step
+
+        This function moves the outlier need_runtime_repack inside the create_model_request
```

Comment:
nit: "this function appends the model needs repack to the create_model_request or create_model_package_request"?

Reply:
Updated